### PR TITLE
chore(forwardToDiscord): add logging

### PR DESCRIPTION
**Pull request by wescopeland**
Adds a bunch of logging to `forwardToDiscord()` to try to debug some of the spurious issues we've been seeing.

### CODE REVIEW

File: app/Community/Listeners/NotifyMessageThreadParticipants.php [hunk 1]
```diff
@@ -74,7 +76,26 @@ private function forwardToDiscord(
...
+
         $message->body = Shortcode::stripAndClamp($message->body, 1850, preserveWhitespace: true);
+        Log::info("[{$messageThread->title}] Discord forward: Message body processed", [
+            'message_id' => $message->id,
+            'thread_id' => $messageThread->id,
+            'from_user' => $userFrom->username,
+            'to_user' => $userTo->username,
+            'original_title' => $messageThread->title,
+            'processed_length' => mb_strlen($message->body),
+            'truncated' => mb_strlen($message->body) === 1850,
```
Comment (Jamiras) [inline app/Community/Listeners/NotifyMessageThreadParticipants.php:96]: This should be 1853. "..." is appended after clamping.

Comment (Jamiras) [reply app/Community/Listeners/NotifyMessageThreadParticipants.php:96]: Or perhaps just ">= 1850"?

Comment (Jamiras) [reply app/Community/Listeners/NotifyMessageThreadParticipants.php:96]: Also applies to copies below

**Review by Jamiras — COMMENTED** [review]
Why not just log the payload and the response?

**Comment by wescopeland** [comment]
> Why not just log the payload and the response?

I was particularly curious if anything in our code was perhaps causing the outgoing data to Discord to be corrupted somehow, and if so, where.

I've cut a lot down in latest.

File: app/Community/Listeners/NotifyMessageThreadParticipants.php [hunk 2]
```diff
@@ -141,6 +143,7 @@ private function forwardToDiscord(
                     //  - Original:  "Unwelcome Concept: Lots of Rings [12345] (Sonic the Hedgehog)"
                     //  - Formatted: "12345: Lots of Rings (Sonic the Hedgehog)"
                     if (preg_match('/^(Incorrect type:|Issue:|Unwelcome Concept:|Writing:)\s*(.*)\s*\[([0-9]+)\]\s*(\(.*\))$/', $messageThread->title, $titleMatches)) {
+                        $originalTitle = $messageThread->title;
```
Comment (Jamiras) [inline app/Community/Listeners/NotifyMessageThreadParticipants.php:146]: This doesn't look to be used.

Comment (wescopeland) [reply app/Community/Listeners/NotifyMessageThreadParticipants.php]: Looks like I missed that line. Removed in latest.

**Review by Jamiras — APPROVED** [review]

**Pull request merged**